### ISSUE TITLE
Unable to install in HACS

### ISSUE THREAD

**Issue by snigehere** [matthewturner/salusfy#31]
Hi Matthew .. 
Many thanks for picking up and solving the issues in this integration ... I have the salus it500 at home and being able to integrate it in home assistant is great.

I wanted to replace the version floringhimie shared with yours but when I try to add it as a custom integration I get an error

<Integration matthewturner/salusfy> Repository structure for v0.6.1 is not compliant 

I am running hacs Integration version:	2.0.1

Do you have an words of wisdom you could share on this? Any options to solve it ... I guess I could manually overwrite the files but that would cause an issue with keeping up to date.

**Comment by Alfamonk** [comment]
Same here, so I can't use it... 

![2025-01-08_095509](https://github.com/user-attachments/assets/51cc4de8-cdb0-4b21-b9ae-ab15bcfe3108)

**Comment by matthewturner** [comment]
Sorry @snigehere, I didn't see this issue until now. When I installed it, I followed floringhimie's instructions and manually added it. I didn't know you could add it through HACS UI.

I've been looking around but I can't find documentation about the repository structure. I'll keep looking and hopefully get it resolved this weekend.

Thanks for your interest!

**Comment by matthewturner** [comment]
Ah found it... I've published v0.7.0 which gets passed that error:

![image](https://github.com/user-attachments/assets/9a8f28dc-ec6a-423e-a1d1-a642f74d0d33)

I have not clicked "Add" yet though so I don't know if there are other issues after this. I'll have a look when I have time.

**Comment by matthewturner** [comment]
Right, I've now clicked add and can see it's overwritten my original implementation and everything seems to work.

There is no UI to configure the climate.yaml so you'll have to still do that manually in a text editor as it shows in the readme.

Let me know if you get stuck but for the moment I'll close this issue.
